exchanges/order/order_types.go

```diff
@@ -306,6 +306,7 @@ const (
 	AnyType
 	Liquidation
 	Trigger
+	OptimalLimitIOC
```

Comment:
Please add support for this with `StringToOrderType` and `func (t Type) String() string {` and add test cases